docs/ui-coverage/configuration/elementfilters.mdx

```diff
@@ -161,3 +153,67 @@ A boolean that represents whether or not a matched element should be included in
 ```
 #name
 ```
+
+---
+
+### Excluding Elements by Attribute
+
+#### Config
+
+```json
+{
+  "elementFilters": [
+    {
+      "selector": "[data-role='decorative']",
+      "include": false
+    }
+  ]
+}
+```
+
+```xml
+<body>
+  <button data-role="decorative">
+    <img src="icon.png" />
+  </button>
+  <button data-role="primary">
+    View
+  </button>
+</body>
+```
+
+#### Elements shown in UI Coverage
+
+```
+[data-role="primary"]
+```
+
+### Excluding dynamic elements by pattern
+
+#### Config
+
+```json
+{
+  "elementFilters": [
+    {
+      "selector": "[class^='auth']",
+      "include": false
+    }
+  ]
+}
+```
+
+#### HTML
+
+```xml
+<body>
+  <iframe class="auth908283794"></iframe>
```

Comment:
this iframe would not be considered interactive anyway, maybe use something with tabindex="0" or a button or something